CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Hosting a Bitcoin Core review club meeting
+
+## General Tips
+
+Some tips for hosting a review club meeting:
+
+- try to post notes and questions on the Friday the week before the meeting.
+  That gives attendees time to review the PR and prepare for the meeting.
+- prepare a range of questions from basic to advanced. The review club is a way
+  to help new developers develop. There should be something for everone.
```

Comment:
To reiterate what @jonatack said, putting in the effort to prep for the meeting will really pay off. I found it useful to have answers to all the questions written down in advanced, especially when there are multiple correct answers and code references to give. A participant may get the answer sorta right, so having text readily available to make use of either directly or as an aid in responding will make the session run smoothly.